src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3215,8 +3223,6 @@ void ApplicationManagerImpl::UnregisterApplication(
     MessageHelper::SendUnsubscribedWayPoints(*this);
   }
 
-  EndNaviServices(app_id);
```

Comment:
Now `EndService` will not be sent for other unregister reasons, such as changing the system language